core/sis-feature/src/test/java/org/apache/sis/feature/builder/AssociationRoleBuilderTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.sis.feature.builder;
+
+import org.apache.sis.util.iso.SimpleInternationalString;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import java.util.Locale;
+import org.apache.sis.feature.DefaultAssociationRole;
+import org.apache.sis.feature.DefaultFeatureType;
+import org.apache.sis.internal.simple.CitationConstant;
+import org.apache.sis.referencing.NamedIdentifier;
+import org.apache.sis.setup.GeometryLibrary;
+import org.apache.sis.util.iso.DefaultNameFactory;
+import org.apache.sis.xml.IdentifierSpace;
+import org.apache.sis.xml.XLink;
+import org.opengis.util.InternationalString;
+
+/**
+ * Unit tests for class {@link AssociationRoleBuilder}.
+ *
+ * @date 2017-09-22
```

Comment:
`@date` is not a standard Javadoc tag as far as I know? We rather use `@version`. I'm not aware of a formal policy about the version number; in Apache SIS I tend to put the SIS version when the file got the last significant change. So in this case it would be `@version 0.8`. You are welcome to add `@author` tag too if you want.

Reply:
You're completely right \@date is not standard Javadoc.
I am going to change this and am going to omit it in future commits too.
Added author tag too.
Thanks.